webpackfile.js

```diff
@@ -3,7 +3,7 @@ const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const HtmlWebpackRootPlugin = require('html-webpack-root-plugin');
 const TSConfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
-
+const TerserPlugin = require('terser-webpack-plugin');
 /**
  * NestJs uses a custom wrapper around require() for allows it to show a
```

Comment:
Not part of this change-set, but I just noticed this grammar error in the comment here :upside_down_face:

Reply:
ah good point. I'll go ahead and fix that in 1a69c85